examples/vision/autoencoder.py

```diff
@@ -0,0 +1,174 @@
+"""
+Title: Convolutional Autoencoder For Image Denoising
+Author: Santiago L. Valdarrama
+Date created: 2021/03/01
+Last modified: 2021/03/01
+Description: How to train an autoencoder that can be used for denoising images.
+"""
+
+"""
+## Introduction
+
+This example illustrates the implementation of a Deep Convolutional Autoecoder that can
+be used to denoise images. This implementation is based on an original blog post from 
+François Chollet.
+"""
```

Comment:
Suggestion:

- Provide a link to @fchollet 's blog post (I assume it's this one https://blog.keras.io/building-autoencoders-in-keras.html)
- Refactor a little (shortening)
- Fix a type in "Autoencoder"
- Add "mapping noisy digits images.... to clean digits images"
- Refer to the MNIST dataset

```suggestion
This example demonstrates how to implementat of a deep convolutional autoencoder
for image denoising—mapping noisy digits images from the MNIST dataset to clean digits
images. This implementation is based on an original blog post titled
[Building Autoencoders in Keras](https://blog.keras.io/building-autoencoders-in-keras.html)
by François Chollet.
"""
```

Reply:
Great, yes, that's the link to the original post.